packages/address/README.md

```diff
@@ -74,6 +74,8 @@ await getTranslationKey('US', 'province'); // => "state"
 await getTranslationKey('JA', 'province'); // => "prefecture"
 await getTranslationKey('CA', 'zip'); // => "postalCode"
 await getTranslationKey('US', 'zip'); // => "zipCode"
+await getTranslationKey('CA', 'address2'); // => "aptUnitNumber"
+await getTranslationKey('CA', 'address2'); // => "aptSuiteEtc"
```

Comment:
You forgot to change the country code. One should be `CA`, the other one could be `FR` for instance.

Reply:
🔥

Reply:
Edited